tcf_website/static/common/sortClasses.js

```diff
@@ -83,6 +102,37 @@ const sortByGpa = () => {
     selectOrderOfSort(sortGpaAsc, "#gpa-sort-btn", "GPA");
 };
 
+const sortByRecency = () => {
+    activeSort = "recency";
+    if (!$("#recency-sort-btn").hasClass("active")) {
+        $("#rating-sort-btn").removeClass("active");
+        $("#diff-sort-btn").removeClass("active");
+        $("#gpa-sort-btn").removeClass("active");
+        $("#rating-sort-btn").html("Rating");
+        $("#diff-sort-btn").html("Difficulty");
+        $("#gpa-sort-btn").html("GPA");
+        $("#recency-sort-btn").addClass("active");
+    } else {
+        sortRecencyAsc *= -1;
+    }
+    selectOrderOfSort(sortRecencyAsc, "#recency-sort-btn", "Last Taught");
+};
+
+const sortByRecencyOnce = () => {
```

Comment:
This function is essentially a copy of `sortByRecency()` - there's probably a way to make the code more concise? One way I can think of is defining this as a two-line function that first sets `sortRecencyAsc = 1` and then immediately calls `sortByRecency()`, which essentially accomplishes the same thing, but that feels kinda scuffed.

Reply:
I tried doing a two-line function, but I ran into problems with sortRecencyAsc not updating to -1 when the last taught sort button was not previously active. I ended up condensing the 2 sortByRecency functions into a single function, but this might've made the code more confusing (like in the way I updated bind-sort.js). I'm not sure if I should use this condensed function or leave the repeated code for readability purposes.